lib/twitter-ads/reach_estimate.rb

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2015 Twitter, Inc.
+
+module TwitterAds
+  module ReachEstimate
+
+    class << self
+
+      # Get a reach estimate for the specified line item details.
+      #
+      # @example
+      #   TwitterAds::ReachEstimate.get_reach_estimate(
+      #     CLIENT, account, "ALL_ON_TWITTER", "PROMOTED_TWEETS", "WEBSITE_CLICKS",
+      #     12, similar_to_followers_of_user: "12", gender: "2")
+      #
+      # @param client [Client] The Client object instance.
+      # @param account [Account] The Ads Account instance for this request.
+      # @param product_type [String] The product type being targeted.
+      # @param objective [String] The objective being targeted.
+      # @param user_id [Long] The ID of the user whose content will be promoted.
+      # @param opts [Hash] A Hash of extended options.
```

Comment:
minor: this will parse our fine when you run `yard` but for readability in code lets insert a newline before the `opts` param.